src/registrar/models/utility/domain_helper.py

```diff
@@ -29,19 +29,19 @@ def validate(cls, domain: str | None, blank_ok=False) -> str:
         if not isinstance(domain, str):
             raise ValueError("Domain name must be a string")
         domain = domain.lower().strip()
-        if domain == "":
-            if blank_ok:
-                return domain
-            else:
-                raise errors.BlankValueError()
+        if domain == "" and not blank_ok:
```

Comment:
ended up making this change to satisfy a lint requirement but let me know if we want to change it back and I can add an exception to this function on the linter